homeassistant/components/light/hue.py

```diff
@@ -330,7 +336,9 @@ def device_info(self):
                 # Philips hue bulb models respond differently to hue/sat
                 # requests, so we convert to XY first to ensure a consistent
                 # color.
-                command['xy'] = color.color_hs_to_xy(*kwargs[ATTR_HS_COLOR])
+                rgb = color.color_hs_to_RGB(*kwargs[ATTR_HS_COLOR])
+                xy_color = self.hue_converter.rgb_to_xy(rgb[0], rgb[1], rgb[2])
+                command['xy'] = xy_color
```

Comment:
Your issue is probably here since we do not actually set `rgb(0,0,255)`. This is due to #13665.

Reply:
Well actually this is not my issue, I tryed using the `hue` and `sat` directly as commands to the hue bridge, but that makes the bulb colors even worse off from what I selected. I am still sending ''xy' color to the bulb just as original, but  the problem I am having lays in the fact that some of my phiplips hue lights can not produce the whole color space that normal "hue" "sat" covers. When selecting a color that lies within the capabilities of the light everything is kind of fine I think and that color is produced by the light (in the current code and with this new PR). 

However when selecting a color that lies outside of the capabilities of the light things go wrong. somehow the closest color needs to be found. Apperently letting the hue bridge decide wich color that schould be gives poor results with a color representing more purple than blue when selecting rgb(0,0,255). When I convert the color using the correct color gamut as specified by the Hue bridge for that light, results are way better and the light actually turns pretty blue for rgb(0,0,255).